### DISCUSSION TITLE
Dropdown max-h value will prevent the bottom dropdown items to be clicked

### DISCUSSION THREAD

**Discussion by esphoenixc** [saadeghi/daisyui#2139] [Q&A]
I gave max-h value to the ul element and the bottom items of the dropdown is not clickable.

I made the dropdown to be shorter and scrollable. When you clicked the <li> items at the bottom, it just closes the dropdown.

Also, when you alt-tab, and come back to where dropdown is open, the element is now clickable.

After adjusting the max-h to shorter from 345px to 250px, it is now clickable.. 

I think the issue is if the dropdown height goes over the certain height, the bottom element is not clickable

**Comment by saadeghi** [comment]
Can you share a code example?

**Reply by esphoenixc** [reply]
As I tested more cases, I found it wasn't about the height or max-h issue, it's a just very weird behavior.

As you see in the picture, the red marked area is where element is not clickable.  Only the red marked area is not clickable. However, when I alt-tab and come back to the page where the dropdown is open, it is now clickable. 

Excuse me for not being able to share the source code.

![image](https://github.com/saadeghi/daisyui/assets/84483515/fe9c46b4-da56-433a-aafa-3961476c780c)

**Comment by esphoenixc** [comment]
` :style="{ visibility: isDropDown ? 'visible' : 'hidden' }"`
 
 This is how I control the visibility of dropdown. And this is causing the issue.

**Reply by saadeghi** [reply]
If you're using the [`<details>` tag](https://daisyui.com/components/dropdown/#method-1-using-details-and-summary-tags) it should be opened/closed using `open` attribute.
